Highrachy/ballers-ws
# #172642472 Fix double scroll bug on index search form

#### What does this PR do?
This PR fixes the double scroll bug on the index page search form 

#### Description of Task to be completed?
This PR fixes the double scroll bug on the index page search form 

#### How should this be manually tested?
* Checkout into this branch
* Visit the home page, you should not see double scroll on the search box select form

#### Screenshots (if appropriate)
##### Before
<img width="1421" alt="Screenshot 2020-05-11 at 7 00 55 PM" src="https://user-images.githubusercontent.com/26963369/81595080-cca7c000-93b9-11ea-86f9-39d209d9c0bd.png">

##### After
<img width="1415" alt="Screenshot 2020-05-11 at 7 00 12 PM" src="https://user-images.githubusercontent.com/26963369/81595106-d4fffb00-93b9-11ea-9d94-b1bd0b978878.png">
